assets/js/components/HypothesisOpener.js

```diff
@@ -203,9 +203,25 @@ module.exports = class HypothesisOpener {
   }
 
   static applyStyleArticleBody($elm) {
-    $elm.style.display = 'block';
-    $elm.style.float = 'right';
-    $elm.style.marginBottom = '48px';
+    annotationPosition();
+    $elm.style.display = 'inline-block';
+    $elm.style.marginBottom = '24px';
+
+    function annotationPosition() {
+      var screenTest = window.innerWidth;
+      if (screenTest < 730) {
+        $elm.style.float = 'none';
+        $elm.style.marginLeft = 'auto';
+        $elm.style.marginRight = 'auto';
+      } else {
+        $elm.style.float = 'right';
+        $elm.style.marginLeft = 'unset';
+        $elm.style.marginRight = 'unset';
+      }
+
+    }
+
+    window.addEventListener('resize', annotationPosition);
```

Comment:
why does any of this need to be in javascript. could we not just add the css?

Reply:
It would probably be better to just toggle classes, but let's not worry about resolving that here.

Reply:
> It would probably be better to just toggle classes, but let's not worry about resolving that here.

I managed to move this to css within a media query